README.md

```diff
@@ -23,17 +23,11 @@ require_once 'vendor/autoload.php';
 
 $client = new \OpenEuropa\CdtClient\ApiClient(
     new \GuzzleHttp\Client(),
-    new \Http\Factory\Guzzle\RequestFactory(),
-    new \Http\Factory\Guzzle\StreamFactory(),
+    new \GuzzleHttp\Psr7\HttpFactory(),
```

Comment:
In the previous version, the code was not working fine. I used `HttpFactory` instead, that implements interfaces both for Request and Stream factories.